src/arrayMethodSort.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const compare =
+      compareFunction ||
+      function compareFunction2(x, y) {
+        return String(x) > String(y);
```

Comment:
Using `String(x) > String(y)` implicitly converts the result to a boolean. Instead, you should use `String(x).localeCompare(String(y))` which returns a number that indicates whether `x` is before, after, or equal to `y` in sort order.